### DISCUSSION TITLE
Is there any plan to add a plugin similar to remote-ssh in vscode?

### DISCUSSION THREAD

**Discussion by ydong08** [pragtical/pragtical#431] [Ideas]

**Comment by jgmdev** [comment]
No plans from my side, but would be a welcome addition if you or some one else develops it.

**Comment by SwissalpS** [comment]
What are the features you would most want such a plugin to provide?

**Comment by ydong08** [comment]
It`s enough for me to edit those source code files which may be in different encoding formats on remote server.

**Comment by PerilousBooklet** [comment]
Adam (from Lite XL) recently began working on a [plugin that allows remote streaming of Lite XL](https://github.com/adamharrison/lite-xl-remote-stream) .
